.github/ISSUE_TEMPLATE/launch.md

```diff
@@ -19,6 +19,8 @@ please update this ticket description to include references to those tickets.
 - [ ] Determine where the starting database will come from.
 - [ ] Determine who manages DNS.
 - [ ] Determine the caching strategy.
+- [ ] Take screenshots of "before" site.
+- [ ] Add site to [web.archive.org](http://web.archive.org/).
```

Comment:
This sounds good, but what was the catalyst for this? Was there a time we did not do this and we ended up wishing we had?

Reply:
Good question. I just found it in the list and carried it over. I presume it was to aid us in creating case studies?